docs/LayoutWithFlexbox.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: basics-layout
+id: layout-with-flexbox
 title: Layout with Flexbox
 layout: docs
 category: The Basics
 permalink: docs/flexbox.html
```

Comment:
I could have sworn these links all worked on my machine. Now they don't, though. Maybe something weird with caching when you keep your docs server running